### ISSUE TITLE
Juggernaut's Blade Dance talent grants unconditional lifesteal

### ISSUE THREAD

**Issue by Undead94100** [ValveSoftware/Dota2-Gameplay#15478]
### Ability name

Blade Dance

### Description

As per title, Juggernaut's level 20 Blade Dance talent seems to grant unconditional 60% lifesteal that is simply tied to the ability rather than granting lifesteal upon crit, which is what I assume was the intention. In the video example below, 2 injured level 30 Juggernauts with the exact items and one of them having no points in Blade Dance, heal themselves to full health in different amounts of attacks, with neither of them using a crit attack once.

https://github.com/ValveSoftware/Dota2-Gameplay/assets/112407737/2d7b9e24-1a17-4348-b3ca-07acd60defb9

### Example Match ID (and possibly Timestamp)

Unavailable, but can be easily reproduced

**Comment by jhgit0219** [comment]
Pretty sure it's just adding 60% lifesteal to the skill itself, the way it's worded doesn't say upon crit.

**Comment by Undead94100** [comment]
That's just a rephrased first sentence of the issue. Here's the problem with this:

1. There is no reason to tie lifesteal to a crit unless the intention was to lifesteal only upon crit. By level 20, all of Juggernaut's skills are maxed out, so he will always benefit from it regardless, meaning the talent might as well just be an internal "+60% lifesteal", similar to talents that Huskar and Drow Ranger have.
2. 60% lifesteal is huge, ability-wise only comparable to Wraith King's Vampiric Spirit with a level 10 talent. However, thanks to Juggernaut's very low BAT and Omnislash/Swiftslash, he can heal much more health than Wraith King can.

TL;DR the talent should either lifesteal only upon crit or be renamed to "+% lifesteal" and reduced for better consistency.

**Comment by Undead94100** [comment]
It appears that as of patch 7.36, Juggernaut's Blade Dance lifesteal talent acts as intended, healing only upon crit.
